CMakeLists.txt

```diff
@@ -49,6 +49,8 @@ if (NOT DEFINED HEADLESS)
     set(HEADLESS TRUE)
 endif()
 
+option(BUILD_ZSTD_SUPPORT "Build zstd support" ON)
```

Comment:
I sent this as an email to you last week, but maybe it went into spam: The consensus internally is that if you pass ZSTD_ROOT for CMake on the command line to the root dir of a ZSTD download, FindZSTD.cmake should find your include and lib and use those in the build.  That's actually what our internal CI does.  So I don't think you need to specify anything related to ZSTD in this PR.